bin/cron/build_contact_rollups

```diff
@@ -0,0 +1,428 @@
+#!/usr/bin/env ruby
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require_relative '../../lib/state_abbr'
+require src_dir 'database'
+require_relative('../../dashboard/config/environment')
+require 'cdo/properties'
+require 'json'
+
+DASHBOARD_REPORTING_DB_READONLY = sequel_connect(
+  CDO.dashboard_reporting_db_reader, CDO.dashboard_reporting_db_reader)
+
+UPDATE_BATCH_SIZE = 100
+
+DASHBOARD_DB_NAME = "dashboard_#{Rails.env}"
```

Comment:
Yes, don't need one. PEGSASUS_DB will choose the correct database name based on environment. The problem is that we are doing cross-DB queries and we need to explicitly name the source database name (Dashboard).